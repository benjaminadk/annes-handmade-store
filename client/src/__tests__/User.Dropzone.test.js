import React from 'react'
import UserDropzone from '../components/User.Dropzone'
import { shallow } from 'enzyme'

test('User Dropzone Component renders correctly', () => {
  const wrapper = shallow(<UserDropzone />)
  expect(wrapper).toMatchSnapshot()
})
