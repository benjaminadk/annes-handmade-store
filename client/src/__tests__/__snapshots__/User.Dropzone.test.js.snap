// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Dropzone Component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(UserDropzone) />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "classes": Object {
        "dropzone": "UserDropzone-dropzone-4",
        "dropzoneContainer": "UserDropzone-dropzoneContainer-3",
        "profileAvatar": "UserDropzone-profileAvatar-2",
        "root": "UserDropzone-root-1",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "classes": Object {
          "dropzone": "UserDropzone-dropzone-4",
          "dropzoneContainer": "UserDropzone-dropzoneContainer-3",
          "profileAvatar": "UserDropzone-profileAvatar-2",
          "root": "UserDropzone-root-1",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
